source/about/self-hosted-subscriptions

```diff
@@ -0,0 +1,172 @@
+====================
+Self-Hosted Subscriptions
+====================
+
+Purchasing a Subscription
+------------------------------------
+
+Open the `Customer Portal page <https://customers.mattermost.com>`__. Enter the required information, and check the box to confirm **I have read and agree to the Privacy Policy**. Then select **Next**. When you've completed the process and validated your email address, you can proceed with the steps below. If you have an existing Customer Portal account, you can log in with your previously created login credentials.  
+
+Once you are logged in to the Customer Portal: 
+
+1. Choose a subscription, and enter the number of users in the **Order summary** field. This indicates the number of users you can have on this subscription's instance. For more information about how users are defined, see our `FAQ <https://mattermost.com/pricing-self-managed/#faq>`__.
+2. (Optional for Enterprise subscriptions) You can add `Premier Support <https://mattermost.com/support/>`__, the cost of which is automatically added to your order total.
+3. Select **Next Step**.
+4. Enter your billing and payment information.
+5. Accept the **Terms**.
+6. Select **Complete**.
+7. Choose **Download the license key**.
+
+.. note::
+If you experience any problems with your transaction, please contact our Support Team via the form provided in the Customer Portal. If possible, keep the error message/number that you received on hand as it may assist with their investigation.
```

Comment:
```suggestion
    If you experience any problems with your transaction, please contact our Support Team via the form provided in the Customer Portal. If possible, keep the error message/number that you received on hand as it may assist with their investigation.
```